packages/nouns-webapp/src/pages/Vote/index.tsx

```diff
@@ -266,10 +268,10 @@ const VotePage = ({
         </div>
         {proposal && proposalActive && (
           <>
-            {showBlockRestriction && !hasVoted && (
+            {proposalCreationTimestamp && showBlockRestriction && !hasVoted && (
               <Alert variant="secondary" className={classes.blockRestrictionAlert}>
-                Only NOUN votes that were self delegated or delegated to another address before
-                block {proposal.createdBlock} are eligible for voting.
+                Only Nouns in your wallet or delegated to you before{' '}
+                {dayjs(proposalCreationTimestamp*1000).format('MMMM D, YYYY h:mm A z')} can vote.
```

Comment:
`nit` could be wrong but would
`daysjs.unix(proposalCreationTimestamp)` maybe be just a tad cleaner?

Reply:
Yea, why not. It would help the readability.